sql/telemetry_derived/asn_aggregates_v1/metadata.yaml

```diff
@@ -9,3 +9,14 @@ owners:
 labels:
   incremental: true
   schedule: daily
+scheduling:
+  dag_name: bqetl_asn_aggregates
+  parameters: ["n_clients:INT64:500"]
+  skip_dry_run: true   # query accesses payload_bytes_raw.telemetry
```

Comment:
The dry run would fail here since it doesn't have permissions to access payload_bytes_raw